production/helm/loki/templates/read/deployment-read.yaml

```
@@ -95,8 +95,18 @@ spec:
           {{- end }}
           securityContext:
             {{- toYaml .Values.loki.containerSecurityContext | nindent 12 }}
+          {{- with .Values.loki.livenessProbe }}
+          livenessProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.loki.readinessProbe }}
           readinessProbe:
-            {{- toYaml .Values.loki.readinessProbe | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.loki.startupProbe }}
+          startupProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
           {{- if .Values.read.livenessProbe }}
           livenessProbe:
             {{- toYaml .Values.read.livenessProbe | nindent 12 }}
```

Just noticed: This is odd, for unknown reasons, loki read has an own livenessProbe.

WDYT?

```suggestion
          {{- with .Values.read.livenessProbe | default .Values.loki.livenessProbe }}
          livenessProbe:
            {{- toYaml . | nindent 12 }}
```